lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/AzureBatchClasspathProvider.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.runime.azbatch;
+
+import org.apache.reef.runtime.common.files.RuntimeClasspathProvider;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Access to the classpath according to the REEF file system standard.
+ */
+public class AzureBatchClasspathProvider implements RuntimeClasspathProvider {
```

Comment:
Minor: it seems the naming of interface implantation is kind of chaotic in Reef. Sometime it uses xxxxxImpl, sometimes it does not. I will post it in the maillist.

Reply:
Right. It seems that all other implementations follow "<RuntimeName>ClasspathProvider" naming scheme, so I'll leave it as is.